src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyle = {};
+  const arrStr = sourceString.split(';');
+
+  for (const pieceArr of arrStr) {
```

Comment:
The task requires using iteration methods like `map` or `filter` instead of loops. Consider refactoring this `for...of` loop to use an appropriate iteration method.